decomposed/extensions/MDDE_LDM/Profile/Model/Methods/mdde_InitializeExportExtension.xml

```diff
@@ -3,25 +3,44 @@
 <a:Name>mdde_InitializeExportExtension</a:Name>
 <a:CreationDate>1701423896</a:CreationDate>
 <a:Creator>WillemOtten</a:Creator>
+<a:Comment>This method is included in the extension to load the contents of templates that are defined in the optionally attached export extension.
+Since it is not possible to directly access the GTL templates in another extension via GTL, the contents of these templates is loaded in local extended attributes so it can be included in the XML export</a:Comment>
 <a:TargetCategory.Type>1</a:TargetCategory.Type>
 <a:MethodScript>Sub %Method%(obj)
-   &#39;Implement your method on &lt;obj&gt; here
-   &#39;Set success indicator to true
+   
    WriteInfo &quot;MDDE LDM Export extension initializion started&quot;
-   obj.setExtendedAttribute cMDDE_EXT_CODE &amp; &quot;.&quot; &amp; cMDDE_EXPORT_EXTENSION_INITIALIZATION_INDICATOR, True
    
    &#39;Set default template and attribute that apply to all object types
    Dim attributeTemplateName: attributeTemplateName = &quot;mddex_NamedObject_Attributes_Export_Template&quot;
    Dim attributeExtendedAttributeName: attributeExtendedAttributeName = &quot;mddex_NamedObject_Attributes_Export_Content&quot;
-   
+   Dim errorsFound: ErrorsFound = False   
+   Dim success: Success = False
    
    &#39;Set the mddex contents attribute on model
-   applyTemplateOnObject obj, obj, &quot;mddex_Model_Export_Template&quot;, &quot;mddex_Model_Export_Content&quot;
-   applyTemplateOnObject obj, obj, attributeTemplateName, attributeExtendedAttributeName
+   success = applyTemplateOnObject (obj, obj, &quot;mddex_Model_Export_Template&quot;, &quot;mddex_Model_Export_Content&quot;)
+   If success = False Then
+      errorsFound = True
+   End If
+   
+   success = applyTemplateOnObject (obj, obj, attributeTemplateName, attributeExtendedAttributeName)
+   If success = False Then
+      errorsFound = True
+   End If
    
    &#39;Process all child objects recursively in all packages
-   processPackage obj, obj, attributeTemplateName, attributeExtendedAttributeName
+   success = processPackage (obj, obj, attributeTemplateName, attributeExtendedAttributeName)
+   If success = False Then
+      errorsFound = True
+   End If
+   
    WriteInfo &quot;MDDE LDM Export extension initializion finished&quot;
+   
+   &#39;Set success indicator to true
+   If errorsFound = True Then
+      Msgbox &quot;There where errors processing the MDDE LDM Export extension, consult the output window for more information.&quot;
```

Comment:
Maybe name it script output window (cause there are 4 output windows I think).

Reply:
Do we want to move this type of call to a generic function?

This way if later we realize using MsgBox is not a great idea, we can refactor it more easily.

Reply:
Did the rename of the  output window to script output but did not refactor it to a generic method, agreed that this is a better solution but maybe not needed for this PR